core/client/app/styles/layouts/users.css

```diff
@@ -108,6 +108,14 @@ a.user-list-item {
     font-size: 12px;
 }
 
+.user-list-item-body .red {
+    display: inline-block;
+    margin-top: 3px;
+    color: var(--red);
+    white-space: nowrap;
+    font-size: 12px;
+}
+
```

Comment:
This is pretty much a duplicate of `.description` - not sure how to correctly display the message in red and keep formatting of `.description`. // @john